pallets/loans/src/lib.rs

```diff
@@ -33,6 +33,7 @@ use frame_support::{
     transactional, PalletId,
 };
 use frame_system::pallet_prelude::*;
+pub use market::{Market, MarketState};
```

Comment:
I don't see the usage of `MarketState` here. Can we remove this import?

Reply:
Removed